examples/Solver_Features/Acoustics/compute_spectrum_production.py

```diff
@@ -274,7 +288,4 @@ def acoustics_output(input_file, output, window, weighting, observer_id):
     plot_oaspl(observer_id_all, oaspl_all, weighting, window)
 
 
-INPUT_FILE = "09caab3a-dc97-4f40-9727-0fe94818cdf1_total_acoustics_v3.csv"
-acoustics_output(
-    INPUT_FILE, output="OASPL.csv", window="Hann", weighting="A", observer_id=-1
-)
+acoustics_output(INPUT_FILE, OUTPUT_FILE, WINDOW, WEIGHTING, OBSERVER_ID)
```

Comment:
@thomas to clarify about the `main()` topic, see below. Nevertheless, no need to change anything - just FYI.

lines 239 & 291 are effectively acting as a `main()` function, but by a different name here. I meant to suggest omitting the `def acoustics_output():` bit all together and instead run the contained code directly.

Instead of:
def acoustics_output(input_file, output, window, weighting, observer_id):
... code using `input_file, output, etc.` ...
acoustics_output(INPUT_FILE, OUTPUT_FILE, WINDOW, WEIGHTING, OBSERVER_ID)

Use this:
... code using `INPUT_FILE, OUTPUT, etc.` ...

Reply:
Understood and modified.